recipes/python/config/access_hosts_api_usecases.py

```diff
@@ -0,0 +1,165 @@
+import sys
+import login.login_api as login_api
+import config.access_hosts_api as access_hosts_api
+
+nbmaster = ""
+username = ""
+password = ""
+domainName = ""
+domainType = ""
+hostName = ""
+
+EXCLUDE_CONFIG_NAME = "Exclude"
+
+def print_disclaimer():
+    print("\n-------------------------------------------------------------------------------------------------")
+    print("--                          This script requires Python3.5 or higher.                          --")
+    print("--    The system where this script is run should have Python 3.5 or higher version installed.  --")
+    print("-------------------------------------------------------------------------------------------------")
+    print("The script requires 'requests' library to make the API calls.")
+    print("You can install the library using the command: pip install requests")
+    print("-------------------------------------------------------------------------------------------------")
+
+def print_usage():
+    print("\nCommand-line usage (should be run from the parent directory of the 'config' directory):")
+    print("\tpython -W ignore -m config.access_hosts_api_usecases -hostName <hostName> -nbmaster <masterServer> -username <username> -password <password> [-domainName <domainName>] [-domainType <domainType>]")
+    print("Note: hostName is the name of the VMware Access host to add/delete using the Access Host APIs.\n")
+    print("-------------------------------------------------------------------------------------------------")
+
+def read_command_line_arguments():
+    if len(sys.argv)%2 == 0:
+        print("\nInvalid command!")
+        print_usage()
+        exit()
+
+    global nbmaster
+    global username
+    global password
+    global domainName
+    global domainType
+    global hostName
+    global workload
+
+    for i in range(1, len(sys.argv), 2):
+        if sys.argv[i] == "-nbmaster":
+            nbmaster = sys.argv[i + 1]
+        elif sys.argv[i] == "-username":
+            username = sys.argv[i + 1]
+        elif sys.argv[i] == "-password":
+            password = sys.argv[i + 1]
+        elif sys.argv[i] == "-domainName":
+            domainName = sys.argv[i + 1]
+        elif sys.argv[i] == "-domainType":
+            domainType = sys.argv[i + 1]
+        elif sys.argv[i] == "-hostName":
+            hostName = sys.argv[i + 1]
+        else:
+            print("\nInvalid command!")
+            print_usage()
+            exit()
+
+    if nbmaster == "":
+        print("Please provide the value for 'nbmaster'\n")
+        exit()
+    elif username == "":
+        print("Please provide the value for 'username'\n")
+        exit()
+    elif password == "":
+        print("Please provide the value for 'password'\n")
+        exit()
+    elif hostName == "":
+        print("Using default hostName to 'test_vmwareAccessHost123'\n")
+        hostName = "test_vmwareAccessHost123"
+
+def parse_access_host_response(response):
+    global nbmaster
```

Comment:
`global` is only necessary when _assigning_ to a variable from a different code block. This function only _uses_ `nbmaster`.